### DISCUSSION TITLE
Complete Mesmerizer Kits now available on Amazon!

### DISCUSSION THREAD

**Discussion by davepl** [PlummersSoftwareLLC/NightDriverStrip#652] [Announcements]
I've made complete kits available on Amazon!  This includes the custom NightDriver PCB, software pre-installed, the 64x32 matrix, the remote control, the remote control transceiver, and the cable required!

You should be able to plug it in and turn it on, pretty much!  And of course you can update it at any time to the most current code here.

https://amzn.to/3MX1Qje

![image](https://github.com/user-attachments/assets/b22679cf-9e12-4e86-91a7-972c35c08234)

**Comment by truckwiz** [comment]
Did you sell all of these?

Thanks'
      Brian

**Comment by davepl** [comment]
I don’t get their reporting - they had 25 in stock, sold 2, and are now out of stock.  But Im looking into it and will update when I know!

- Dave

**Reply by NH-RedAnt** [reply]
I recently found your youtube channel and have been tinkering with similar hardware and software projects. 

I have a variety of leds in strip, matrix, form. As well as esp32 and  RPi hardware. 

I am also interested in your amazon kit. I haven’t used a HUB75 display yet. But have  a waveshare 96x48 flexible display in my cart.  Went to confirm your board would fit and found you offered kits at some point.

I would also be interested in just your esp32 board, from any source.

**Comment by Unclerojelio** [comment]
Yeah, I had one on order only to have the order canceled a week later. Amazon seems to have quite a bit of latency in their inventory tracking. I’ll look forward to playing the lottery again when the next batch is up. -- RogerOn Oct 5, 2024, at 11:30, David W Plummer ***@***.***> wrote:﻿
I don’t get their reporting - they had 25 in stock, sold 2, and are now out of stock.  But Im looking into it and will update when I know!



- Dave

**Comment by prschguy1** [comment]
Hello Dave, wanted to ask if this was a last run push of this system, or will additional kits be made available later?

Thanks, Craig

**Comment by Thermotronica** [comment]
is there any way to make these larger and smaller? that way we advertise the product on them. we will have to afford one first though...

thx

**Comment by heidepiek** [comment]
How did it turn out with amazon and the 25 boards?  When I read that dave had put boards on amazon, I was thinking about ordering one. But I made a mesmerizer myself with a lolin and homemade pcb so I had not ordered it (https://github.com/PlummersSoftwareLLC/NightDriverStrip/discussions/571). But a day later I wanted to order the original anyway, and by then they were already sold out! So I am also curious if boards will become available again. I know the nightdriver software can be used for many boards, but the mesmerizer is something specific. It does not work on all boards, so a piece of hardware to use the software is desirable! And this discussion started with the title (Complete Mesmerizer Kits now available on Amazon!) So it would be nice to have that.  But still no response from team nightdriver! Too bad. 😞

**Comment by davepl** [comment]
Well, Amazon lost BOTH shipments that we sent them, so I'm out 50 kits!  In the meantime, to make them available, I've put some on eBay for you!

https://www.ebay.com/itm/335713383751

**Reply by heidepiek** [reply]
That’s quite a blow for you, Dave, with Amazon losing 50 of those Mesmerizer sets just like that! I hope you get compensated for it. I see you’ve got some sets listed on eBay, but it says the seller probably doesn’t ship to the Netherlands. The U.S. option is the only one available as well. €129 is quite a steep price, and I assume shipping costs would be added on top of that. And those are no joke – you can count on a hefty €50 at least!

Thanks for getting back to us, although I do think your response came a bit late. I understand this situation is unpleasant for you, but you could have reached out a bit sooner. I would have really liked to own an original Mesmerizer, but I’ll make do with my DIY PCB.

Harrie
